src/l500/l500-private.h

```diff
@@ -57,7 +57,8 @@ namespace librealsense
             DPT_INTRINSICS_FULL_GET     = 0x7F,
             RGB_INTRINSIC_GET           = 0x81,
             RGB_EXTRINSIC_GET           = 0x82,
-            FALL_DETECT_ENABLE          = 0x9D  // Enable (by default) free-fall sensor shutoff (0=disable; 1=enable)
+            FALL_DETECT_ENABLE          = 0x9D, // Enable (by default) free-fall sensor shutoff (0=disable; 1=enable)
+            HW_SYNC_EX_TRIGGER          = 0x19, // Enable (not default) HW sync; will disable freefall
```

Comment:
Move the line before 0x1E to keep the list sorted for maintainability

Reply:
Done